### ISSUE TITLE
Custom change form template breaks mass edit

### ISSUE THREAD

**Issue by andreynovikov** [burke-software/django-mass-edit#82]
I wonder what is the point in this line:
https://github.com/burke-software/django-mass-edit/blob/f69a639fac9912f355726bf051793acb843bdaf8/massadmin/massadmin.py#L189

If I define custom change form for my model admin, mass edit becomes completely broken. If I remove this line from `massadmin.py` everything returns to normal.

**Comment by daris** [comment]
Same problem here, is it possible to somehow workaround this?

**Comment by daris** [comment]
My quick workaround for this issue is to set change_form_template inside overridden render_change_form method instead of in ModelAdmin class directly (so this template will be used only for change form and not for mass edit form):

```
    def render_change_form(self, *args, **kwargs):
        self.change_form_template = 'custom_change_form.html'
        return super().render_change_form(*args, **kwargs)
```

**Comment by PetrDlouhy** [comment]
Is this issue solved by MR #96?

**Comment by PetrDlouhy** [comment]
I am closing this issue as my last question remained without answer.

If the issue still persist in current version of `django-mass-edit`, feel free to reopen this issue, but please define what `completely broken` exactly means.
